docs/usage.md

```diff
@@ -399,30 +297,75 @@ This is especially useful for datasets that grow over time.
 !!! important
     Note again that the virtual Zarr approach requires the same chunking and encoding across datasets. This including when appending to an existing Icechunk-backed Zarr store. See the [FAQ](faq.md#can-my-specific-data-be-virtualized) for more details.
 
-```python
-session = repo.writeable_session("main")
-
+```python exec="on" session="usage" source="material-block" result="code"
 # write the virtual dataset to the session with the IcechunkStore
+session = repo.writable_session("main")
 vds2.vz.to_icechunk(session.store, append_dim="time")
-session.commit("Appended second dataset")
+snapshot_id = session.commit("Appended second dataset")
+print(snapshot_id)
+```
+
+See the [Icechunk documentation](https://icechunk.io/en/latest/virtual/) for more details.
+
+### Writing to Kerchunk's format and reading data via fsspec
+
+The [kerchunk library](https://github.com/fsspec/kerchunk) has its own [specification](https://fsspec.github.io/kerchunk/spec.html) for serializing virtual datasets as a JSON file or Parquet directory.
+
+To write out all the references in the virtual dataset as a single kerchunk-compliant JSON or parquet file, you can use the [virtualizarr.VirtualiZarrDatasetAccessor.to_kerchunk][] accessor method.
+
+```python
+combined_vds.vz.to_kerchunk('output/combined.json', format='json')
+```
+
+These zarr-like references can now be interpreted by [fsspec](https://github.com/fsspec/filesystem_spec), using kerchunk's built-in xarray backend (kerchunk must be installed to use `engine='kerchunk'`).
+
+```python
+combined_ds = xr.open_dataset('output/combined.json', engine="kerchunk")
+print(combined_ds)
```

Comment:
For the dataset used in this usage guide, it is not advisable to write it to a Kerchunk JSON because one of the two bytesbytescodecs will get cut off. I am tempted to just remove this section, as mentioned in slack. We could later add a section that loads a Kerchunk JSON from object storage, which is a more likely use case.